group-coordinator/src/main/java/org/apache/kafka/coordinator/group/runtime/MultiThreadedEventProcessor.java

```diff
@@ -39,7 +40,7 @@ public class MultiThreadedEventProcessor implements CoordinatorEventProcessor {
     /**
      * The accumulator.
      */
-    private final EventAccumulator<Integer, CoordinatorEvent> accumulator;
```

Comment:
Is this changed from integer to topic partition so that we can use different coordinator state partitions (ie consumer offsets vs transactional state)

Reply:
Right. It was also just easier to use TopicPartition everywhere.